src/http-api/index.js

```diff
@@ -0,0 +1,37 @@
+var Hapi = require('hapi')
+var IPFS = require('../ipfs-core')
+var debug = require('debug')
+var log = debug('api')
+
+exports = module.exports
+
+exports.start = function (callback) {
+  // start IPFS and exports.ipfs = new IPFS()
+
+  exports.ipfs = new IPFS()
+
+  var server = exports.server = new Hapi.Server({
+    connections: {
+      routes: {
+        cors: true
+      }
+    }
+  })
+
+  server.connection({
+    port: 9000
+  })
+
+  // load routes
+  require('./routes/version.js')
```

Comment:
why is that _"much"_ cleaner? If there was more than one route on that file I would have to go and

```JavaScript
server.route(require('./routes/version.js').routeA)
server.route(require('./routes/version.js').routeB)
// etc
```

Meaning that I have to alter the route file and index.js, the way I have, I only have to open and modify one file. Also, having the server object available means I always have server methods at my hand.